ydb/docs/en/core/maintenance/manual/backup-collections.md

```diff
@@ -0,0 +1,311 @@
+# Backup Collections Operations
+
+This guide covers all practical operations for creating, managing, and restoring from backup collections. For conceptual information, see [Backup collections concepts](../../concepts/backup-collections.md).
+
+## Creating backup collections {#creating-collections}
+
+### SQL Syntax Reference {#sql-syntax-reference}
+
+```sql
+-- CREATE BACKUP COLLECTION syntax
+CREATE BACKUP COLLECTION `collection_name`
+    ( TABLE `table_path` [, TABLE `table_path` ...] )
+WITH ( STORAGE = 'cluster', INCREMENTAL_BACKUP_ENABLED = 'true' );
+
+-- BACKUP syntax
+BACKUP `collection_name` [INCREMENTAL];
+
+-- DROP BACKUP COLLECTION syntax
+DROP BACKUP COLLECTION `collection_name`;
+
+-- RESTORE syntax
+
+RESTORE `collection_name`
+```
+
+For detailed syntax, see [YQL reference documentation](../../yql/reference/syntax/index.md).
+
+### Basic collection creation {#basic-collection-creation}
+
+Create a backup collection using SQL commands. The collection defines which tables to include and storage settings:
+
+```sql
+CREATE BACKUP COLLECTION `shop_backups`
+    ( TABLE `/Root/shop/orders`, TABLE `/Root/shop/products` )
+WITH ( STORAGE = 'cluster', INCREMENTAL_BACKUP_ENABLED = 'true' );
+```
+
+**Parameters:**
+
+- `STORAGE` - Storage backend for backups:
+  - `'cluster'` - Store backups within the YDB cluster (currently the only supported option)
+  - Future versions may support: `'object-storage'`, `'nfs'`, or other storage backends
+
+- `INCREMENTAL_BACKUP_ENABLED` - Enable incremental backup support:
+  - `'true'` - Enable incremental backups (recommended for production)
+  - `'false'` - Only full backups allowed
+
+## Taking backups {#taking-backups}
+
+### Initial full backup {#initial-full-backup}
+
+After creating a collection, take the initial full backup:
+
+```sql
+BACKUP `shop_backups`;
```

Comment:
Clarify whether these queries are idempotent or not, is it safe to retry many times if you're getting timeouts or something similar?